adiachenko/src/components/Category/index.tsx

```diff
@@ -33,21 +35,57 @@ const CategoryUl = styled.ul`
   }
 `;
 
-const Category = () => (
-  <CategoryUl>
-    <li className="active">
-      ALL
-    </li>
-    <li>
-      DOCUMENTARY
-    </li>
-    <li>
-      COMEDY
-    </li>
-    <li>
-      CRIME
-    </li>
-  </CategoryUl>
-);
+const categories = [
+    {
+        type: 'All'
+    },
+    {
+        type: 'Documentary'
+    },
+    {
+        type: 'Comedy'
+    },
+    {
+        type: 'Crime'
+    }
+];
+
+interface CategoryLiProps {
+    type: string,
+    category: string,
+    setCategory: () => void
+}
+
+const CategoryLi = ({type, category,  setCategory}: CategoryLiProps) => {
+    const isActive = useMemo(() => type.toLowerCase() === category.toLowerCase(), [type, category]);
```

Comment:
I would say that comparing 2 variables is not an expensive operation worth `useMemo()` usage. In fact, the overhead from using extra wrapping function may even do the opposite - consume extra memory. You may consider wrapping some other (preferably, expensive and often invoked) operation, like `.sort()`.